library/ui-styles/src/main/res/values/styles_edit_text.xml

```diff
@@ -11,4 +11,14 @@
         <item name="android:textColor">?vctr_message_text_color</item>
     </style>
 
-</resources>
\ No newline at end of file
+    <style name="Widget.Vector.EditText.Composer.RichTextComposer" parent="Widget.AppCompat.EditText">
```

Comment:
```suggestion
    <style name="Widget.Vector.EditText.RichTextComposer" parent="Widget.AppCompat.EditText">
```

?

Reply:
Sorry, I changed it to the correct value, and also corrected the references to this style in the views.